exercises/practice/dnd-character/tests/Tests.elm

```diff
@@ -0,0 +1,223 @@
+                    \() ->
+                        DndCharacter.modifier 17
+                            |> Expect.equal 3
+            , skip <|
+                test "ability modifier for score 18 is +4" <|
+                    \() ->
+                        DndCharacter.modifier 18
+                            |> Expect.equal 4
+            ]
+        , describe "ability"
+            [ skip <|
+                fuzzWith { runs = 1000, distribution = Test.noDistribution }
+                    (Fuzz.fromGenerator DndCharacter.ability)
+                    "generated ability should be at least 3"
+                <|
+                    \ability -> Expect.atLeast 3 ability
+            , skip <|
+                fuzzWith { runs = 1000, distribution = Test.noDistribution }
+                    (Fuzz.fromGenerator DndCharacter.ability)
+                    "generated ability should be at most 18"
+                <|
+                    \ability -> Expect.atMost 18 ability
+            , skip <|
+                fuzzWith
+                    { runs = 10000
+                    , distribution =
+                        Test.expectDistribution
+                            [ ( Test.Distribution.moreThanZero, "is 3", (==) 3 ) ]
+                    }
+                    (Fuzz.fromGenerator DndCharacter.ability)
+                    "generated ability should sometimes be 3"
+                <|
+                    \_ -> Expect.pass
+            , skip <|
+                fuzzWith
+                    { runs = 1000
+                    , distribution =
+                        Test.expectDistribution
+                            [ ( Test.Distribution.moreThanZero, "is 18", (==) 18 ) ]
+                    }
+                    (Fuzz.fromGenerator DndCharacter.ability)
+                    "generated ability should sometimes be 18"
+                <|
+                    \_ -> Expect.pass
+            , skip <|
+                fuzzWith
+                    { runs = 10000
+                    , distribution =
+                        Test.expectDistribution
+                            [ ( Test.Distribution.atLeast 10, "is 13", (==) 13 )
+                            , ( Test.Distribution.atLeast 85, "is not 13", (/=) 13 )
+                            ]
+                    }
+                    (Fuzz.fromGenerator DndCharacter.ability)
+                    "13 has an approximate 13% chance of being picked"
+                <|
+                    \_ -> Expect.pass
+            ]
+        , describe "character"
+            [ skip <|
+                fuzzWith { runs = 100, distribution = Test.noDistribution }
+                    (Fuzz.fromGenerator DndCharacter.character)
+                    "generated character strength should be within range"
+                <|
+                    \character ->
+                        Expect.all [ Expect.atLeast 3, Expect.atMost 18 ] character.strength
```

Comment:
This is checking that the generated abilities are within range. I did one test per ability for clarity.

An error for a test of this type looks like
```
Given { charisma = 16, constitution = 12, dexterity = 14, hitpoints = 11, intelligence = 12, strength = 21, wisdom = 15 }

    21
    ╷
    │ Expect.atMost
    ╵
    18
```